index.js

```diff
@@ -26,24 +28,26 @@ $.ajax({
     type: "GET"
 });
 
-
-function fillingOfVoids(obj) {
-    if (obj) {
-        return obj;
-    } else {
-        obj = 'Unknown';
-        return obj;
+class Book {
+    constructor(src) {
+        this.data = [src.authors, src.title, src.categories, src.publisher, src.publishedDate];
     }
+    fillingModal() {
+        let content = '';
+        this.data.forEach((item, i, arr) => {
+            if (item === undefined) {
+                item = 'Unknown';
+            };
+            content += TICKETS[i] + item;
+        })
+        return content;
+    };
 };
 
-
 function modalContent(src) {
-    document.querySelector('#bodyModal').innerHTML =
-        'Author: ' + fillingOfVoids(src.authors) + '<br>Title: ' +
-        fillingOfVoids(src.title) + '<br>Category: ' +
-        fillingOfVoids(src.categories) + '<br>Publisher: ' +
-        fillingOfVoids(src.publisher) + '<br>Published date: ' +
-        fillingOfVoids(src.publishedDate);
+    document.querySelector('.modal-header').innerHTML = `${MODAL_HEADER}${src.title}</h4>`;
+    let book = new Book(src);
```

Comment:
If you have fixed all above, please change this part. items will contain array of Books, use method find to find Book with id === src.id